awesome.md

```diff
@@ -0,0 +1,31 @@
+# Awesome CRI-O [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+This list follows the [awesome
+manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md), and
+collects every useful resource about CRI-O.
+
+## Contents
+
+- [Resources](#resources)
+  - [Articles](#articles)
+  - [Community](#community)
+  - [Videos](#videos)
+  - [Websites](#websites)
+
+## Resources
+
+### Articles
+
+- [CentOS 7 with CRI-O on EKS (Nov 20, 2018)](https://medium.com/errnothxbye/centos-7-with-cri-o-on-eks-ae9684aff764)
+
+### Community
+
+- [Kubernetes Slack #cri-o](https://slack.com/app_redirect?channel=CAZH62UR1)
```

Comment:
There's also the cri-o channel on freenode. It's mostly spam from a bot but some folks reach out on freenode.

Reply:
Added it but I cannot link it to somewhere. Do you have any suggestions?

Reply:
Maybe [https://webchat.freenode.net/](https://webchat.freenode.net/)?